docs/features/routing.rst

```diff
@@ -236,3 +236,51 @@ Ocelot will also allow you to put query string parameters in the UpstreamPathTem
 
 In this example Ocelot will only match requests that have a matching url path and the query string starts with unitId=something. You can have other queries after this
 but you must start with the matching parameter. Also Ocelot will swap the {unitId} parameter from the query string and use it in the downstream request path. 
+
+Security Options
+^^^^^^^^^^^^^^^^
+
+Ocelot allows you to manage multiple patterns for allowed/blocked IPs using the `IPAddressRange <https://github.com/jsakamoto/ipaddressrange>`_ package.

```

Comment:
We must provide brief licensing information.

Reply:
MPL-2.0 Link added